utils/lib.sh

```diff
@@ -499,6 +499,18 @@ service_is_available() {
 # golang
 # ------
 
+# Architecture detection
+
+if [[ $(uname -m) == x86_64* ]]; then
```

Comment:
Nit: Is the command `arch` not availalbe? To me, the command name is more descriptive.

Reply:
It's not available on all Linux distributions. For instance, on ArchLinux the command `arch` doesn't exist.

Reply:
True @unixfox